internal/oci/runtime_vm.go

```diff
@@ -141,6 +141,7 @@ func (r *runtimeVM) CreateContainer(c *Container, cgroupParent string) (err erro
 
 	select {
 	case err = <-createdCh:
+		f.Close()
```

Comment:
how is the file closed in the timeout case?

Reply:
I think similar handling should be done in timeout case.
Should I include that change in this PR ?

Reply:
yeah can you include it in this PR?

Reply:
I have updated the PR